### PR TITLE
Added openHAB theme fragment for Paper UI.

### PR DESCRIPTION
**Pull request by dnobel**
Signed-off-by: Dennis Nobel <dennis@dnobel.de>

### CODE REVIEW

File: addons/ui/org.openhab.ui.paperui/.settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -0,0 +1,100 @@
+eclipse.preferences.version=1
```
Comment (kaikreuzer) [inline addons/ui/org.openhab.ui.paperui/.settings/org.eclipse.jdt.core.prefs]: Do we need all these entries in this file?

Comment (dnobel) [inline addons/ui/org.openhab.ui.paperui/.settings/org.eclipse.jdt.core.prefs]: I guess not. Will fix it.

**Comment by dnobel** [comment]
Well, the fragment needs the host bundle (org.eclipse.smarthome.ui.paperui) in the target platform. Not sure, if this is possible. But that´s why the build fails.

**Comment by kaikreuzer** [comment]
No worries, I'll take care of the build. Could you please squash your commits?

**Comment by dnobel** [comment]
Done.

**Pull request merged**